pandas/core/arrays/string_.py

```diff
@@ -96,7 +103,7 @@ def __from_arrow__(
         return StringArray._concat_same_type(results)
 
 
-class StringArray(PandasArray):
+class StringArray(PandasArray, ObjectStringArrayMixin):
```

Comment:
ObjectStringArrayMixin is already mixed in to PandasArray, is it needed again here?